frontend/Auth/ResetPassword.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import { Button, ButtonKind } from "../Styleguide/Button";
+import { FormField } from "../Styleguide/FormField";
+import { FormFieldLabel } from "../Styleguide/FormFieldLabel";
+import { Input } from "../Styleguide/Input";
+import { RouterProps } from "react-router";
+import { useMutation } from "react-query";
+import { flaxFetch } from "../Utils/flaxFetch";
+import { unary } from "lodash-es";
+
+export function ResetPassword(props: RouterProps) {
+  const [resetPasswordFormData, setResetPasswordFormData] =
+    useState<ResetPasswordFormData>({
+      email: "",
+    });
+
+  const history = useHistory();
+
+  const submitMutation = useMutation<
+    ResetPasswordResultData,
+    Error,
+    React.FormEvent<HTMLFormElement>
+  >(
+    (evt) => {
+      evt.preventDefault();
+      let requestPromise = flaxFetch<ResetPasswordResultData>(
+        `/send-reset-password-email`,
+        {
```

Comment:
Let's provide the abort signal to fetch, as explained in https://react-query.tanstack.com/guides/query-cancellation#_top. Cancelation of queries is a best practice and can prevent some bugs and performance issues

Reply:
Thank you! Was not able to implement signal in useMutation.